.github/workflows/main.yml

```diff
@@ -169,6 +169,7 @@ jobs:
           create-args: >-
             mamba
             python=${{ matrix.python-version }}
+            lmoments3
```

Comment:
Perhaps we can have a specific build that installs lmoments3. I can look into implementing that.

Reply:
This has been addressed.